src/macros.rs

```diff
@@ -100,7 +100,7 @@ macro_rules! body {
         $crate::init_panic_handler();
 
         #[allow(unused_mut)]
-        let mut r = |$gc: &mut $crate::Runtime| $code;
+        let mut r = |#[allow(unused_variables)] $gc: &mut $crate::Runtime| $code;
```

Comment:
Now that I see the ew version, is this lambda still required? you can just expand $code directly at the end, right?

Reply:
In the other PR, that's true. This one will be merged before the other interop one so it still needs this.